snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedReceivedSnapshot.java

```diff
@@ -31,6 +31,7 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(FileBasedReceivedSnapshot.class);
   private static final boolean FAILED = false;
   private static final boolean SUCCESS = true;
+  private static final int BLOCK_SIZE = 1 << 19;
```

Comment:
Is that 512Kb? :smile: I always have an easier time reading `512 * 1024`, but idk if that's a strong argument for it.

Reply:
That's fair, I'll change it